querydsl-core/src/main/java/com/querydsl/core/types/ConstantImpl.java

```diff
@@ -28,32 +28,34 @@
 
     private static final int CACHE_SIZE = 256;
 
-    @SuppressWarnings({"rawtypes", "unchecked"}) //generic array creation not possible
-    private static final Constant<Character>[] CHARACTERS = new Constant[CACHE_SIZE];
+    private static final Constant<Boolean> FALSE = new ConstantImpl<Boolean>(Boolean.FALSE);
```

Comment:
Using `create(false)` would utilize the cache instead of creating another instance, right?